src/lib.rs

```diff
@@ -202,9 +206,17 @@ pub mod algorithms;
 pub mod errors;
 /// Supported padding schemes.
 pub mod padding;
-/// RSASSA-PKCS1-v1_5 Signature support
+/// PKCS#1 v1.5 support as described in [RFC8017 § 8.2].
+///
+/// [RFC8017 § 8.2]: https://datatracker.ietf.org/doc/html/rfc8017#section-8.2
 pub mod pkcs1v15;
-/// RSASSA-PSS Signature support
+/// Support for the [Probabilistic Signature Scheme] (PSS) designed by
+/// Mihir Bellare and Phillip Rogaway, a.k.a. RSASSA-PSS.
+///
+/// Specified in [RFC3447 § 8.1].
+///
+/// [Probabilistic Signature Scheme]: https://en.wikipedia.org/wiki/Probabilistic_signature_scheme
+/// [RFC8017 § 8.1]: https://datatracker.ietf.org/doc/html/rfc8017#section-8.1
 pub mod pss;
```

Comment:
Might be worth moving all of these comments to be toplevel rustdoc comments on the respective modules, rather than defined inline here.

Reply:
Oops, just noticed this comment cites the old RFC too

Reply:
No strong opinion. Maybe it's worth adding mentions of PKCS#1 v1.5, PSS and OAEP to the top-level description (so that the developer can see what's supported), while keeping such details and links here, in the inline comments.

Reply:
rustdoc automatically puts the first line in the toplevel description, regardless of where it’s defined

Reply:
I meant something like:
```diff
-//! RSA Implementation in pure Rust.
+//! RSA Implementation in pure Rust. It supports several schemes described in RFC 8017:
+//! - PKCS#1 v1.5 encryption scheme
+//! - OAEP encryption scheme
+//! - PKCS#1 v1.5 signature scheme
+//! - PSS signature scheme
 //!
+//! These schemes are described below.
 //!
 //! # Usage
 //!
```

Reply:
Sure, that works.

Reply:
Added that and moved the other comments to the tops of the respective modules.